src/ralph/accounts/models.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+from dj.choices import (
+    Country,
+    Gender,
+)
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class RalphUser(AbstractUser):
+
+    gender = models.PositiveIntegerField(
+        verbose_name=_('Gender'),
```

Comment:
stick to one convention in verbose names (' or ", lower-case or title-case (first letter)) - I'll suggest ' and lower-case